WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -278,12 +278,19 @@ private void ConfigureStateMachine(CoinJoinManager coinJoinManager)
 			});
 	}
 
+	private void UpdateCountDown()
+	{
+		ElapsedTime = $"{DateTime.Now - _countDownStarted:mm\\:ss}";
+		RemainingTime = $"-{_autoStartTime - DateTime.Now:mm\\:ss}";
```

Comment:
This assumes the `ElapsedTime` and the `RemainingTime`  will never be greater that 59:59 what is wrong. Basically these values are computed from from a configuration setting on the server so, they should be treated as "user inputs" (or admin inputs) what means that you cannot assume anything or that you have to validate the value before using it.

As a side note, in most of the code base we use `DateTimeOffset` instead of `DateTime` because in some cases the timezone is important. To be completely honest, in most cases we don't need the timezone but it is important to know the timezone is there because once we strip it by using DateTime then some operations can give unexpected results.